From: SeongJae Park <sj@kernel.org>
Subject: mm/damon: add kernel-doc comment for damon_operations->cleanup_target
Date: Tue, 15 Jul 2025 11:52:38 -0700

Commit 769bd386e490 ("mm/damon/core: add cleanup_target() ops callback")
on mm-unstable added cleanup_target field to damon_operations, but forgot
adding the kernel-doc description.  This causes a build warning.  Fix it
by adding the description.

Link: https://lkml.kernel.org/r/20250715185239.89152-2-sj@kernel.org
Signed-off-by: SeongJae Park <sj@kernel.org>
Reported-by: Stephen Rothwell <sfr@canb.auug.org.au>
Closes: https://lore.kernel.org/20250715185549.264260b8@canb.auug.org.au
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 include/linux/damon.h |    1 +
 1 file changed, 1 insertion(+)

--- a/include/linux/damon.h~mm-damon-core-add-cleanup_target-ops-callback-fix
+++ a/include/linux/damon.h
@@ -576,6 +576,7 @@ enum damon_ops_id {
  * @get_scheme_score:		Get the score of a region for a scheme.
  * @apply_scheme:		Apply a DAMON-based operation scheme.
  * @target_valid:		Determine if the target is valid.
+ * @cleanup_target:		Clean up each target before deallocation.
  * @cleanup:			Clean up the context.
  *
  * DAMON can be extended for various address spaces and usages.  For this,
_
